### PR TITLE
[Comments] Add comments to Proto files

### LINKED ISSUE #2567: Improve comments for all `proto` files
Add at least a liner description for each message, some `proto` files didn't add it 

image

https://hewigovens.github.io/wc-docc/documentation/walletcore/

### PR DESCRIPTION
**Pull request by catenocrypt**
## Description

Add more complete comments to Proto files.  Fixes #2567.

## How to test

Check src/proto/*.proto files.

## Types of changes

* Comment only

## Checklist

- [x] Create pull request as draft initially, unless its complete.
- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.
- [x] If there is a related Issue, mention it in the description.

If you're adding a new blockchain

- [ ] I have read the [guidelines](https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria) for adding a new blockchain.

### CODE REVIEW

File: src/proto/Aeternity.proto [hunk 1]
```diff
@@ -11,8 +11,10 @@ message SigningInput {
     // Address of the recipient with "ak_" prefix
     string to_address = 2;
 
+    // Amount, as uint256
```
Comment (hewigovens) [inline src/proto/Aeternity.proto:14]: Let's add this comment to all number represented in bytes
```suggestion
    // Amount (uint256, serialized little endian)
```

File: src/proto/Algorand.proto [hunk 2]
```diff
@@ -3,46 +3,65 @@ syntax = "proto3";
 package TW.Algorand.Proto;
 option java_package = "wallet.core.jni.proto";
 
+// Simple transfer message, transfer an amount to an address
 message Transfer {
+    // Destination address (string)
```
Comment (hewigovens) [inline src/proto/Algorand.proto:8]: the type itself is `string` already
```suggestion
    // Destination address
```

File: src/proto/Algorand.proto [hunk 3]
```diff
@@ -3,46 +3,65 @@ syntax = "proto3";
...
     string to_address = 1;
+
+    // Amount
     uint64 amount = 2;
 }
 
+// Asset Transfer message, with assetID
 message AssetTransfer {
+    // Destination address (string)
```
Comment (hewigovens) [inline src/proto/Algorand.proto:17]:
```suggestion
    // Destination address
```

File: src/proto/Binance.proto [hunk 4]
```diff
@@ -181,20 +179,32 @@ message TimeRelockOrder {
...
+    // Chain ID
     string chain_id = 1;
+
+    // Source account number
     int64 account_number = 2;
+
+    // Sequence number (account specific)
     int64 sequence = 3;
+
     int64 source = 4;
```
Comment (hewigovens) [inline src/proto/Binance.proto:199]: `source` field is missing: https://github.com/bnb-chain/BEPs/blob/master/BEP10.md

File: src/proto/Bitcoin.proto [hunk 5]
```diff
@@ -33,7 +34,7 @@ message TransactionInput {
 
 // Bitcoin transaction out-point reference.
 message OutPoint {
-    // The hash of the referenced transaction.
+    // The hash of the referenced transaction (reverse byte order).
```
Comment (hewigovens) [inline src/proto/Bitcoin.proto:37]:
```suggestion
    // The hash of the referenced transaction (network byte order, usually needs to be reversed).
```

File: src/proto/Cosmos.proto [hunk 6]
```diff
@@ -22,12 +30,14 @@ message Height {
     uint64 revision_height = 2;
 }
 
+// Sync/asyn mode for transaction broadcast
```
Comment (hewigovens) [inline src/proto/Cosmos.proto:33]:
```suggestion
// Transaction broadcast mode
```

File: src/proto/Cosmos.proto [hunk 7]
```diff
@@ -281,30 +292,43 @@ message Message {
     }
 }
 
+// Options for transaction encoding: Json (older) or Protobuf.
```
Comment (hewigovens) [inline src/proto/Cosmos.proto:295]:
```suggestion
// Options for transaction encoding: Amino (older) or Protobuf.
```

File: src/proto/Ethereum.proto [hunk 8]
```diff
@@ -44,14 +48,16 @@ message Transaction {
...
         string from = 1;
 
+        // Destination address
         string to = 2;
 
         // ID of the token (256-bit number)
         bytes token_id = 3;
 
-        // The amount of tokens being transferred
+        // The amount of tokens being transferred (256-bit number)
```
Comment (hewigovens) [inline src/proto/Ethereum.proto:60]:
```suggestion
        // The amount of tokens being transferred (uint256, serialized little endian)
```

File: src/proto/Waves.proto [hunk 9]
```diff
@@ -34,17 +50,24 @@ message CancelLeaseMessage {
...
     }
 }
 
-// Transaction signing output.
+// Result containing the signed and encoded transaction.
 message SigningOutput {
+    // signature data
     bytes signature = 1;
+
+    // transaction in Json format
```
Comment (hewigovens) [inline src/proto/Waves.proto:70]:
```suggestion
    // transaction in JSON format
```

File: src/proto/Ripple.proto [hunk 10]
```diff
@@ -7,27 +7,35 @@ import "Common.proto";
...
+    // Ledger sequence number
     int32 last_ledger_sequence = 4;
 
+    // Source account
     string account = 5;
 
+    // Target account
     string destination = 6;
 
     int64 destination_tag = 7;
```
Comment (hewigovens) [inline src/proto/Ripple.proto:28]:
```suggestion
    // A Destination Tag
    int64 destination_tag = 7;
```

File: src/proto/Ripple.proto [hunk 11]
```diff
@@ -7,27 +7,35 @@ import "Common.proto";
...
 
     int64 flags = 8;
```
Comment (hewigovens) [inline src/proto/Ripple.proto:30]:
```suggestion
    // Transaction flags, optional
    int64 flags = 8;
```

**Review by hewigovens — COMMENTED** [review]
Nice work, I did go through all the files

**Review by Milerius — DISMISSED** [review]
Good job, some minor typos

**Review by Milerius — APPROVED** [review]

**Pull request merged**